### ISSUE TITLE
[Adaptive Cards - Expense Report]: Screen Reader is conveying the incorrect role information as Button for the 'Export to PDF' control.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#97]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
Adaptive Card Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Expense Report'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate all the controls under 'Visual'.
8. Navigate to 'Export to PDF' link and observe the issue.

**Actual Result:**
While navigating to the 'Export to PDF' button, Talkback is announcing incorrect role as 'Button'. As on double tap on it a new web page is getting open.

**'Expected Result:'**
Talkback should announce the correct role as 'Link' for the 'Export to PDF' and when the focus lands on it then screen reader should announce as 'Export to PDF' link.

**User Impact:**
This issue will impact the screen reader users as they will get confused and treat the control as 'Button'.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/1ae52b11-7bd6-4c30-9845-4fa1f651f391

![Talkback is announcing incorrect role info](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/5b361b88-1464-4722-bf96-c87a816f1e88)

_No response_

### Card JSON

```JSON
MA
```

**Comment by vagpt** [comment]
This issue is fixed partially as of now it is announcing twice role hence created a new bug #201
